crates/ruma-events/src/call/member.rs

```diff
@@ -257,7 +257,7 @@ mod tests {
                 call_id: "123456".to_owned(),
                 scope: CallScope::Room,
             }),
-            device_id: "ABCDE".to_owned(),
+            device_id: device_id!("ABCDE").to_owned(),
```

Comment:
There is an `owned_device_id!` macro:

```suggestion
            device_id: owned_device_id!("ABCDE"),
```

And the same below.

Reply:
updated all occurrences.